src/v2/Apps/_Preferences2/preferencesRoutes.tsx

```diff
@@ -0,0 +1,16 @@
+import loadable from "@loadable/component"
+import { AppRouteConfig } from "v2/System/Router/Route"
+
+const PreferencesApp = loadable(
+  () => import(/* webpackChunkName: "aboutBundle" */ "./PreferencesApp"),
```

Comment:
We'll want to update the `webpackChunkName` prop to `preferencesBundle -- this is how webpack knows to split the js here into a new file that _only_ contains code related to the preference center

Reply:
Updated.